lib/electric_slide/call_queue.rb

```diff
@@ -276,11 +276,9 @@ def bridge_agent(agent, queued_call)
         remove_agent agent
       end
 
-      agent.call.on_joined do 
-        agent.callback :connect, self, agent.call, queued_call
-      end
-
       agent.join queued_call
+      agent.callback :connect, self, agent.call, queued_call
```

Comment:
Nope, just did it this way for consistency with call_agent. Will switch back to a callback as there's a reason for it.